xds/internal/xdsclient/transport/transport.go

```diff
@@ -105,7 +106,14 @@ type Transport struct {
 // cause the transport layer to send an ACK to the management server. A non-nil
 // error is returned from this function when the data model layer believes
 // otherwise, and this will cause the transport layer to send a NACK.
-type UpdateHandlerFunc func(update ResourceUpdate) error
+type OnRecvHandlerFunc func(update ResourceUpdate) error
+
+// OnSendHandlerFunc is the implementation at the xDS data model layer, which
```

Comment:
Hmm... I don't think this is technically the data model layer where this is handled. Maybe just say authority instead?

Reply:
done